aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -196,8 +223,8 @@ export const ShippingExample = () => {
           pad={{ horizontal: 'xxsmall' }}
         >
           <Form
-            validate="blur"
             value={formValues}
+            // messages={{ required: requiredErrorMessage }}
```

Comment:
was there a reason for adding this if its below ?

Reply:
whoops comment should we taken out.